convert/convert_gemma2.go

```diff
@@ -34,10 +34,20 @@ func (p *gemma2Model) KV(t *Tokenizer) llm.KV {
 }
 
 func (p *gemma2Model) Replacements() []string {
-	return append(
-		p.gemmaModel.Replacements(),
+	return []string{
```

Comment:
site note.. why isn't this a map?

Reply:
The tensor names follow a pattern, but they aren't _exact_ matches. The way we did this before was w/ a map and a regex, which worked fine but was more code. The string replacer is more compact, but you have to do some gymnastics in how you think about the tensor names.